### PR TITLE
Jmoiron/master

### PR DESCRIPTION
**Pull request by ebenoist**

### CODE REVIEW

**Review by cgrubbs — APPROVED** [review]
Nice!

**Review by wenn — APPROVED** [review]
Thanks!

**Comment by ebenoist** [comment]
np

File: go.mod [hunk 1]
```diff
@@ -0,0 +1,10 @@
+module github.com/jmoiron/sqlx
```
Comment (cgrubbs) [inline go.mod:1]: @ebenoist wondering if this go.mod file needs surgery, I'm getting the following error when trying to pull in this new version:

`[cg/sync_active_users][~/dev/feed]$ go get -u "github.com/reverbdotcom/sqlx"
go: downloading github.com/reverbdotcom/sqlx v0.0.0-20210511173003-f5462c8dec5c
go get: github.com/reverbdotcom/sqlx@v0.0.0-20180110223232-8f6781503779 updating to
	github.com/reverbdotcom/sqlx@v0.0.0-20210511173003-f5462c8dec5c: parsing go.mod:
	module declares its path as: github.com/jmoiron/sqlx
	        but was required as: github.com/reverbdotcom/sqlx`

Comment (ebenoist) [reply go.mod:1]: Almost certainly. I might have missed some of those edits in the rebase. Feel free to wire that back "correctly"

**Pull request merged**